INCIDENTS.md

```diff
@@ -0,0 +1,51 @@
+# Incident Response Protocol
+
+
+## What's an incident?
+
+An incident is defined here as a severe immediate or short-term potential degredation in the user-facing performance of our services.
```

Comment:
I think that the definition should remain light-touch, but is there anything we can put here that will help us take appropriate actions earlier - thinking of the fact that the incident that prompted this doc took several hours to notice because we all assumed that it was a minor issue only affecting users behind a particular toggle.

Automated testing helps, but there will always be something we haven't tested for - maybe we could say something about bare minimum investigations of bugs before we decide if they are/aren't an incident? Not sure

Reply:
Could it be turning this into a set of questions?

* Who is affected?
* How has it affected their service needs?
* Are their workarounds we could communicate?
* What comms would help mitigate interference?

After answering these questions you could do a quick review with someone to assess the damage?

Reply:
Added some questions to ask when a potential incident arises.